plugins/org.eclipse.epsilon.dt.exeed/src/org/eclipse/epsilon/dt/exeed/ExeedImageTextProvider.java

```diff
@@ -89,13 +90,25 @@ public void setShowStructuralInfo(boolean showStructuralInfo) {
 
 	private ImageDescriptor getImageDescriptor(String icon) {
 		ImageDescriptor imageDescriptor = null;
-		
-		imageDescriptor = plugin.getImageDescriptor("icons/" + icon + ".gif");
-		if (imageDescriptor == null) {
-			imageDescriptor = plugin.getImageDescriptor("icons/" + icon + ".png");
+		// icon might be a platform:/resource uri
+		if (icon != null) {
+			if (icon.startsWith ("platform:/")) {
+				try {
+					URI iconuri = URI.createURI(icon, false);
+					imageDescriptor = plugin.getImageDescriptor(iconuri);
+				}
+				catch (IllegalArgumentException ex) {
+					// icon is not a valid uri

```

Comment:
The exception should be logged using LogUtil.log(e);

Reply:
Fixed